.gitignore

```diff
@@ -1,6 +1,2 @@
 node_modules/
-working
-.DS_Store
-views/
-data/
-mockups/
+
```

Comment:
Pls retain folders/files  working/ , .DS_Store, views/ , data/ , mockups/ . in the gitignore file.